tq/transfer_queue.go

```diff
@@ -633,7 +634,12 @@ func (q *TransferQueue) run() {
 
 // canRetry returns whether or not the given error "err" is retriable.
 func (q *TransferQueue) canRetry(err error) bool {
-	return errors.IsRetriableError(err)
+	switch cause := errors.Cause(err).(type) {
```

Comment:
Rad ⚡️. I know that both the `switch (...).(type)` is a legit way to do this, but I think the convention in the majority of LFS looks something like:

```go
func (q *TransferQueue) canRetry(err error) bool {
        err = errors.Cause(err)

        if cause, ok := err.(*url.Error); ok {
                return cause.Temporary() || cause.Timeout()
        }
        return errors.IsRetriableError(err)
}
```